plugins/scaffolder-annotator-action/README.md

```diff
@@ -0,0 +1,110 @@
+# Annotator custom action for Scaffolder Backstage
+
+The annotator module for [@backstage/plugin-scaffolder-backend](https://www.npmjs.com/package/@backstage/plugin-scaffolder-backend).
+
+## Getting started
+
+### Installing on the new backend system
+
+To install the module into the [new backend system](https://backstage.io/docs/backend-system/), add the following into the `packages/backend/src/index.ts` file:
+
+```ts title="packages/backend/src/index.ts
+const backend = createBackend();
+
+// highlight-add-start
+backend.add(
+  import('@janus-idp/backstage-scaffolder-backend-module-annotator/alpha'),
+);
+// highlight-add-end
+
+backend.start();
+```
+
+### Installing on the legacy backend system
+
+1. Install the Annotator custom action plugin using the following command:
```

Comment:
Maybe add a section on top that explains what this custom action package provides overall? Like all the features and then the specific actions for timestamping and scafolded-from.